spec/wakes_spec.rb

```diff
@@ -129,22 +176,39 @@ class WakeableModel < ActiveRecord::Base
       create(:location, :path => '/legacy-location-four', :canonical => false, :resource => resource_two)
     end
 
+    let(:resource_three) { create(:resource) }
+    let!(:canonical_location_three) do
+      create(:location, :path => '/canonical-location-three', :resource => resource_three)
+    end
+    let!(:legacy_location_five) do
+      create(:location,
+             :host => 'solidjoys.desiringgod.org',
+             :path => '/legacy-location-five',
+             :canonical => false,
+             :resource => resource_three)
+    end
+
     describe '::create_redis_graph' do
       before do
         # clear out anything that got set up during initialization
         Wakes::REDIS.del Wakes::REDIS.keys
         resource_one.update_attribute(:legacy_paths_in_redis, nil)
         resource_two.update_attribute(:legacy_paths_in_redis, nil)
-      end
+        resource_three.update_attribute(:legacy_paths_in_redis, nil)
 
-      it 'stores the entire wakes redirector graph into redis' do
         Wakes.create_redis_graph
+      end
 
+      it 'stores the entire (local) wakes redirector graph into redis' do
         expect(Wakes::REDIS.get('/legacy-location-one')).to eq('/canonical-location-one')
         expect(Wakes::REDIS.get('/legacy-location-two')).to eq('/canonical-location-one')
         expect(Wakes::REDIS.get('/legacy-location-three')).to eq('/canonical-location-two')
         expect(Wakes::REDIS.get('/legacy-location-four')).to eq('/canonical-location-two')
       end
+
+      it 'doesn\'t store paths from other hosts in redis' do
```

Comment:
I would either use double quotes so you don't have to escape the single quote OR expand the contraction to "does not"